nipype/interfaces/spm/preprocess.py

```diff
@@ -497,20 +497,24 @@ def _list_outputs(self):
                 outputs['normalized_files'] = self.inputs.apply_to_files
             outputs['normalized_source'] = self.inputs.source
         elif 'write' in self.inputs.jobtype:
+            if isdefined(self.inputs.write_preserve) and self.inputs.write_preserve:
+                prefixNorm = ''.join(['m', self.inputs.out_prefix])
```

Comment:
one quick question, just to ensure i understand this correctly. so if write_preserve is set, then the output prefix doesn't matter?